docs/dev/raft-in-scylla.md

```diff
@@ -4,18 +4,19 @@ Date: 2022-05-06
 
 When we began implementing Raft we wanted to create a reusable and
 well tested component which we could utilize for data, schema and topology
-operations. This is why the Raft library in raft/ has the only dependency
-- on seastar. It provides its own design documents and a README.
+operations. This is why the Raft library in raft/ has the only dependency -
```

Comment:
In particular, the dash at the beginning of the line made it show as a bullet list item (which was seemingly not the intention). Moving it to the end of previous line fixes that.